std/conv.d

```diff
@@ -1241,49 +1238,35 @@ body
 
         do
         {
-            static if (radix == 0)
-            {
-                div = cast(S)(mValue / runtimeRadix );
-                mod = cast(ubyte)(mValue % runtimeRadix);
-                mod += mod < 10 ? '0' : baseChar - 10;
-            }
-            else static if (radix > 10)
-            {
-                div = cast(S)(mValue / radix );
-                mod = cast(ubyte)(mValue % radix);
-                mod += mod < 10 ? '0' : baseChar - 10;
-            }
-            else
-            {
-                div = cast(S)(mValue / radix);
-                mod = mValue % radix + '0';
-            }
+            div = cast(S)(mValue / runtimeRadix );
```

Comment:
The static information about radix is dealt with a few lines down in the switch statement. Note that I did not change any of the unit tests for toImpl().